.github/workflows/deployment-ci.yml

```
@@ -15,7 +15,7 @@ jobs:
   build:
     if: |
       !contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.pull_request.title, '[ci skip]')
-      && !(github.repository == 'kordlib/kord' && github.event_name == 'pull_request')
+      && !(github.event_name == 'pull_request' && github.event.pull_request.repo == 'kordlib/kord')
```

```suggestion
      && !(github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == 'kordlib/kord')
```

This should be what we want here (found this [here](https://github.community/t/get-the-head-repository-name-in-prs/17669)).